src/neptune_scale/exceptions.py

```diff
@@ -435,10 +435,6 @@ class NeptuneSeriesStepNonIncreasing(NeptuneScaleError):
 {h1}
 NeptuneSeriesStepNonIncreasing: Subsequent steps of a series must be increasing.
```

Comment:
As per Siddhant's suggestion

```suggestion
NeptuneSeriesStepNonIncreasing: Subsequent steps of a series must be strictly increasing.
```